spec/system/rental_creation_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+describe 'creating a new rental' do
+  before :each do
+    @tomorrow = Time.zone.tomorrow
+    @day_after = @tomorrow + 1.day
+  end
+
+  describe 'form autopopulation' do
+    before :each do
+      FactoryBot.create :user
+      visit '/rentals/new'
```

Comment:
You should just be able to call create :user as the FactoryBot methods are included in the rails_helper.